conftest.py

```diff
@@ -53,3 +55,17 @@ def user() -> dict:
         "zipcode": "12345",
         "city": "example-city"
     }
+
+
+"""Global helper functions."""
+
+
+def pdfToText(pdf) -> str:
+    """Convert pdf bytes into text."""
+    reader = PyPDF2.PdfFileReader(io.BytesIO(pdf))
```

Comment:
As I worked yesterday with `PyPDF2` I learned that in the current version (2.8.1) `PdfFileReader` is deprecated and `PdfReader` should be used. Which version do we have here?

Reply:
That should be solved after a rebase. Due to the conflict, I need to rebase anyway.